programs/data-credits/src/instructions/update_in_use_data_credits_v0.rs

```diff
@@ -0,0 +1,26 @@
+use crate::state::*;
+use anchor_lang::prelude::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct UpdateInUseDataCreditsArgsV0 {
+  new_owner: Pubkey,
+}
+
+#[derive(Accounts)]
+#[instruction(args: UpdateInUseDataCreditsArgsV0)]
+pub struct UpdateInUseDataCreditsV0<'info> {
```

Comment:
Just rm this for now, we don't need this endpoint. In use data credits are about to change anyway

Reply:
Done